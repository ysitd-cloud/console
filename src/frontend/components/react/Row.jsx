import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

export default function Row({ children, className }) {
  const cls = classNames(className, 'layout', 'row');
  return <div className={cls}>{children}</div>;
}

Row.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};

Row.defaultProps = {
  children: null,
  className: '',
};
