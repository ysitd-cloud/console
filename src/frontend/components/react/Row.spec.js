import React from 'react';
import renderer from 'react-test-renderer';
import Row from './Row';

/* globals describe, expect, it */

describe('React Row Component', () => {
  it('match snapshot', () => {
    const component = renderer.create(<Row>foo</Row>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
