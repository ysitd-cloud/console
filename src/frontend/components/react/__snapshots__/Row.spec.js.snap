// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Row Component match snapshot 1`] = `
<div
  className="layout row"
>
  foo
</div>
`;
